convert2rhel/pkghandler.py

```diff
@@ -365,6 +366,18 @@ def preserve_only_rhel_kernel():
     return
 
 
+def install_gpg_keys():
+    loggerinst = logging.getLogger(__name__)
+    gpg_path = os.path.realpath(os.path.join(os.path.dirname(__file__),
+                                "data/version-independent/gpg-keys/*"))
```

Comment:
I wonder if this works, I'll test it out shortly. But if I get it right, the gpg keys are moved to /usr/share/convert2rhel/ folder during the RPM installation (you can use `utils.data_dir` to get this dir here).

Reply:
Seems to work if we update tests to have a mocked directory. Commiting a change now